### ISSUE TITLE
Suitable w_hm3.snplist for AFR?

### ISSUE THREAD

**Issue by beckse** [bulik/ldsc#407]
Hi! I've come across a LD reference panel for AFR. But I need an appropriate SNP list for --merge-alleles. Does anyone know how to generate something suitable? Thank you!

**Comment by samreenzafer** [comment]
Can I ask where you find it? I need a ref panel for AFR population also for using with MTAG (which relies on LDSC).

**Comment by laleoarrow** [comment]
Also need the w_hm3.snplist file for non-EUR population. Any resources on this matter is highly appreciated!

**Comment by aksarkar** [comment]
@leoarrow1 Use the MAP files from HapMap 3 for your population of interest to get the variants to include https://www.broadinstitute.org/files/shared/mpg/hapmap3/hapmap3_r1_b36_fwd.qc.poly.tar.bz2

If you want superpopulations (for example, AFR), take the union of the constituent population MAP files.

The population codes are given at https://www.broadinstitute.org/medical-and-population-genetics/hapmap-3

You will need to lift over positions/IDs from b36 to match your reference LD matrices.

You will then need to compute the weights yourself as (non-partitioned) LD scores from your matched reference panel.

**Comment by yuj23** [comment]
> @leoarrow1 Use the MAP files from HapMap 3 for your population of interest to get the variants to include https://www.broadinstitute.org/files/shared/mpg/hapmap3/hapmap3_r1_b36_fwd.qc.poly.tar.bz2
> 
> If you want superpopulations (for example, AFR), take the union of the constituent population MAP files.
> 
> The population codes are given at https://www.broadinstitute.org/medical-and-population-genetics/hapmap-3
> 
> You will need to lift over positions/IDs from b36 to match your reference LD matrices.
> 
> You will then need to compute the weights yourself as (non-partitioned) LD scores from your matched reference panel.

Do you know how to get those snps in w_hm3.snplist? The ldscore in w_ld_eur/ does not fully match the w_hm3.snplist list.  Is w_hm3.snplist only for EUR population?

**Comment by laleoarrow** [comment]
@yuj23 According to this #425 issue, it seems that `w_hm3.snplist` is not EUR specific and can be applied to other ancestry.

**Comment by aksarkar** [comment]
@yuj23 From Bulik-Sullivan et al. 2015: "In order to restrict to a set of common, well-imputed variants, we retained only those SNPs in the HapMap 3 reference panel for the LD Score regression."

These would likely be https://www.broadinstitute.org/files/shared/mpg/hapmap3/hapmap3_r1_b36_fwd_consensus.qc.poly.recode.map.bz2 which are consensus calls over all HapMap subpopulations.

**Comment by steniorj** [comment]
To attack this problem I downloaded dataset from HapMap3 (.map and .ped files), gathered the AFR populations, used PLINK to extract a freq file and then awk to make a format similar to the tutorial file

**Comment by wusiwei2022** [comment]
> [@yuj23](https://github.com/yuj23) From Bulik-Sullivan et al. 2015: "In order to restrict to a set of common, well-imputed variants, we retained only those SNPs in the HapMap 3 reference panel for the LD Score regression."
> 
> These would likely be https://www.broadinstitute.org/files/shared/mpg/hapmap3/hapmap3_r1_b36_fwd_consensus.qc.poly.recode.map.bz2 which are consensus calls over all HapMap subpopulations.

@aksarkar 
Hi aksarkar,
What I feel confusing is it seems to me that the authors calculate the ld scores first, and then include only hapmap3 SNPs in ld scoe regression? Therefore, I guess we don't really need to compute LD scores including only SNPs listed in hm3.snp.list? However, the authors must have done inclusion and exclusion on the SNPs in 1000 genome data because the SNPs in the LD score file is much less than the SNPs in the 1000 genome data. I'm wondering what they have done before computing the LD scores.

**Comment by beckse** [comment]
BTW, here is a comment from Raymond Walters saying that w_hm3.snplist is not EUR-specific and may be used with any ancestry. https://groups.google.com/g/ldsc_users/c/q367fQIOVwA/m/Lzs3xMcsCgAJ

**Comment by aksarkar** [comment]
@wusiwei2022 No, you must calculate LD scores both ways, once using all SNPs in 1KG (in order to get the correct LD scores at variants used in the regression, `ref_ld_chr`) and once using only SNPs in HapMap (in order to get the correct weights for variants used in the regression, `w_ld_chr`).
